docs/rest-api.yml

```diff
@@ -17,38 +17,53 @@ securityDefinitions:
-      users:activity!user=username: Update a single user's activity (example horizontal filter).
-      read:users: Read-only access to users' models apart from servers, tokens and authentication state.
-      read:users!user=username: As above limited to a specific user (example horizontal filter).
-      read:users:name: Read-only access to users' names.
-      read:users:roles: Read-only access to a list of users' roles names.
-      read:users:groups: Read-only access to a list of users' group names.
-      read:users:activity: Read-only access to users' activity.
-      read:users:activity!group=groupname: Read-only access to specific group's users' activity (example horizontal filter).
-      admin:users:servers: Grants read, start/stop, create and delete permissions to users' servers and their state.
-      admin:users:server_state: Grants access to servers' state only.
-      users:servers: Allows for starting/stopping users' servers in addition to read access to their models. Does not include the server state.
-      users:servers!server=servername: Limits the above to a specific server (example horizontal filter).
-      read:users:servers: Read-only access to users' server models. Does not include the server state.
-      users:tokens: Grants read, write, create and delete permissions to users' tokens.
-      read:users:tokens: Read-only access to users' tokens.
-      admin:groups: Grants read, write, create and delete access to groups.
-      groups: Grants read and write permissions to groups, including adding/removing users to/from groups.
-      groups!group=groupname: As above limited to a specific group only (example horizontal filter)
-      read:groups: Read-only access to groups.
-      read:services: Read-only access to service models.
-      read:services:name: Read-only access to service names.
-      read:services:roles: Read-only access to a list of service roles names.
-      read:hub: Read-only access to detailed information about JupyterHub.
-      proxy: Allows for obtaining information about the proxy's routing table, for syncing the Hub with proxy and notifying the Hub about a new proxy.
-      shutdown: Grants access to shutdown the Hub.
+    scopes: # Generated based on scope table in jupyterhub/scopes.py
+      (no_scope): Identify the owner of the requesting entity.
+      self:
+        The user’s own resources _(metascope for users, resolves to (no_scope)
+        for services)_
+      all: Everything that the token-owning entity can access _(metascope for tokens)_
+      admin:users:
+        Read, write, create and delete users and their authentication state,
+        not including their servers or tokens.
+      admin:auth_state: Read a user’s authentication state.
+      users:
+        Read and write permissions to user models (excluding servers, tokens
+        and authentication state).
+      read:users:
+        Read user models (excluding including servers, tokens and authentication
```

Comment:
`excluding` or  `including`?

Reply:
Excluding; access to servers, tokens and auth_state info is not granted with the `read:users` scope